docs/skill-guide.md

```diff
@@ -0,0 +1,230 @@
+The <a href="../scaffolding/">scaffolding tool</a> allows you create the folder structure required for a skill.
+
+!!!	Note
+	Before developing your first skill, please read the <a href="../skill/">skill guide</a>.
+
+## Step 1: Setup
+
+Ensure, you have followed the <a href="../quickstart/">preliminaries</a>. We will first create an agent and add a scaffold skill, which we call `my_search`:
+
+``` bash
+aea create my_agent && cd my_agent
+aea scaffold skill my_search
+```
+
+## Step 2: Develop a Behaviour
+
+A `Behaviour` class contains the business logic specific to initial actions initiated by the agent rather than reactions to other events.
+
+In this example, we implement a simple search behaviour. Each time, `act()` gets called, we will send a search request to the OEF.
```

Comment:
It is not mentioned where the code below should be added. At this point of the documentation, is the user aware where this code should be? Worth reminding it?